postgis/Dockerfile

```diff
@@ -1 +1,9 @@
-FROM postgis/postgis:latest
+FROM postgis/postgis:16-master
+
+RUN apt-get update && \
+    apt-get install -y postgresql-16-periods git build-essential postgresql-server-dev-16
+
+WORKDIR /root
+RUN git clone https://github.com/sraoss/pg_ivm.git
+WORKDIR /root/pg_ivm
+RUN make && make install
```

Comment:
Ug - I hate to do it, but I'm installing a non-core postgres extension. It's worth it. 

This allows you, provided you're willing to use a subset of the SQL syntax, to create materialized views that stay up to date in an incremental fashion. Meaning, if a row of a logically parent table is updated, /just/ the affected rows in the materialized view are updated. It's the best of both worlds; the dynamism of a view with the performance and indexability of a materialized view. 

I'm using it to pull the expensive part (joining against locations) out of my crashes view.

:magic_wand: This is black-magic, and it would preclude us from using RDS to host our database, but it's pretty swank.

That said, I think I could implement this on a per-table basis with triggers, but never in this super-general-solution that this extension provides. 

FWIW, this extension was updated with a new release just this month; it's not some dead-on-the-vine thing like one might find out there.